README.md

```diff
@@ -47,10 +47,10 @@ gulp
 ```
 
 ## Contributing
-Refer to [Contibuting Docuemnent]('./docs/CONTRIBUTING.MD) to contribute to this project.
+Refer to [Contibuting Docuemnent](./docs/CONTRIBUTING.MD) to contribute to this project.
```

Comment:
Typo error in` [Contibuting Document]`. Meant to be `[Contributing Document]`. @andela-rekemezie